x/rollapp/types/message_create_rollapp.go

```diff
@@ -46,12 +48,15 @@ func (msg *MsgCreateRollapp) GetSignBytes() []byte {
 func (msg *MsgCreateRollapp) ValidateBasic() error {
 	_, err := sdk.AccAddressFromBech32(msg.Creator)
 	if err != nil {
-		return sdkerrors.Wrapf(sdkerrors.ErrInvalidAddress, "invalid creator address (%s)", err)
-	}
-	if msg.GetMaxSequencers() == 0 {
```

Comment:
why remove this check?
currently `maxSequencer = 0` makes it impossible to register sequencers.

if we want it to be limitless, we need to change sequencer logic as well

Reply:
i updated  sequencer logic to skip num sequencers check when maxSequencer is 0

Reply:
I wonder if we want some optional type or *int instead of overloading 0?
(Not just here but in general)
I don't really have a strong opinion, I think overloading can be fine too